src/constraints/outage_constraints.jl

```diff
@@ -342,8 +349,32 @@ function add_MG_storage_dispatch_constraints(m,p)
             m[:dvMGDischargeFromStorage][s, tz, ts] <= p.s.storage.attr["ElectricStorage"].max_kw * m[:binMGStorageUsed]
         )
     end
-end
 
+    # Future development could make this an option by adding bool inputs an updating load balance/battery dispatch constraints to include these flows
+    if "ElectricStorage" in p.s.storage.types.dc_coupled
+        # Don't let AC coupled elec techs charge DC coupled battery. 
+        for ts in 1:p.s.site.min_resil_time_steps
```

Comment:
So, if you have a run with wind (AC production) and PV that is dc-coupled with storage, the wind cannot charge the battery? Do you have some test cases in this realm in runtests?

Reply:
Yes that's the current set up. Could add an input bool to allow this though. Noting this as a test to add.